plugins/.common

```diff
@@ -12,6 +13,7 @@ add_github_ssh_host() {
 }
 
 install_binary() {
+    [ -n "${EXPERIMENTAL_PLUGIN}" ] && PREFIX=${PREFIX}-experimental
```

Comment:
Might be preferable to do something like this:

```suggestion
	local install_prefix="${PREFIX}"
    if [ -n "${EXPERIMENTAL_PLUGIN}" ]; then
    	install_prefix="${PREFIX}-experimental"
   	fi
```

Then change line 19 from `${PREFIX}` to `${install_prefix}` that way we can avoid changing global variables